streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala

```diff
@@ -105,13 +105,13 @@ class ReceiverSuite extends TestSuiteBase with TimeLimits with Serializable {
     assert(executor.errors.head.eq(exception))
 
     // Verify restarting actually stops and starts the receiver
-    receiver.restart("restarting", null, 100)
-    eventually(timeout(50 millis), interval(10 millis)) {
+    receiver.restart("restarting", null, 600)
```

Comment:
If these timeout bumps fix the flakiness, we should also consider enabling the "block generator throttling" test below (it was disabled via https://github.com/apache/spark/commit/b69c4f9b2e8544f1b178db2aefbcaa166f76cb7a)

Reply:
yeah. lets fix one at a time.